README.md

```diff
@@ -384,6 +384,32 @@ The schemas mirror the CDF API as closely as possible.
 | `metadata`           | `map(string, string)` | Yes       |
 | `dataSetId`          | `long`                | Yes       |
 
+### Sequences
```

Comment:
Link those in the table of contents as well, similar to `- [Asset Hierarchy](#asset-hierarchy)`